nautobot/utilities/testing/views.py

```diff
@@ -125,6 +125,10 @@ def test_get_object_anonymous(self):
             self.client.logout()
             response = self.client.get(self._get_queryset().first().get_absolute_url())
             self.assertHttpStatus(response, 200)
+            response_body = response.content.decode(response.charset)
+            self.assertIn(
+                "/login/?next=" + self._get_queryset().first().get_absolute_url(), response_body, msg=response_body
+            )
```

Comment:
Should we have a corresponding `assertNotIn` added to `test_get_object_without_permission()`?